src/index.js

```diff
@@ -0,0 +1,10 @@
+const { Router } = require('./router');
+const { ArticleList, ArticleInfo } = require('./components');
+
+const router = new Router('app')
+  .add('/articles/:topic', ArticleList)
+  .add('/article/:topic/:id', ArticleInfo)
+  .init();
+
+button1.addEventListener('click', () => router.push('/articles/books'));

```

Comment:
o_O? I need to click a button to open the list?

Reply:
> components can be break to smaller + can be separated to containers and presentation

done

Reply:
done